src/main/java/io/github/thebusybiscuit/exoticgarden/items/Crook.java

```diff
@@ -24,6 +26,11 @@ public Crook(Category category, SlimefunItemStack item, RecipeType recipeType, I
         super(category, item, recipeType, recipe);
     }
 
+    @Override
+    public void preRegister() {
+        addItemHandler((ItemUseHandler) PlayerRightClickEvent::cancel);
```

Comment:
This will literally break the Crook as a whole.
**NEVER** Override preRegister() without calling the super method, because you just removed the BlockBreakHandler by doing that.